src/dos/cdrom_image.cpp

```diff
@@ -771,8 +771,8 @@ bool CDROM_Interface_Image::PlayAudioSector(const uint32_t start, uint32_t len)
 	 *  64-bit.
 	 */
 	player.playedTrackFrames = 0;
-	player.totalTrackFrames = ceil_udivide(track_rate * player.totalRedbookFrames,
-	                                      REDBOOK_FRAMES_PER_SECOND);
+	player.totalTrackFrames = player.totalRedbookFrames *
+	                          (track_rate / REDBOOK_FRAMES_PER_SECOND);
```

Comment:
I am a little bit confused about this change - comment above suggested we are safe from the overflow. Should the comment be updated (or maybe even removed)?

Reply:
The comment was wrong - I must have previously thought both inputs to `ceil_udivide` were 64-bit (at the point when I wrote the comment); but at some point they became 32-bit which is when the rollover risk came into play.

Given the comment is now misleading (as overflow is no longer possible), I'll remove it.  Thanks for catching this!

Reply:
Updated inbound.